hammer_cli_foreman_puppet.gemspec

```diff
@@ -18,5 +18,5 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.test_files    = Dir['{test}/**/*']
 
-  # spec.add_dependency 'hammer_cli_foreman', '>= 2.0.0', '< 3.0.0'
+  spec.add_dependency 'hammer_cli_foreman', '>= 3.0.0', '< 4.0.0'
```

Comment:
I'd set it to `'>= 2.0.0', '< 4.0.0'`

Reply:
Hasn't hammer-cli been bumped at least in nightly yet?